source/management_api/initdb.js

```diff
@@ -20,7 +20,7 @@ var cipher = require('./cipher');
 
 var dirName = !process.pkg ? __dirname : path.dirname(process.execPath);
 var configFile = path.join(dirName, 'management_api.toml');
-var sampleServiceFile = path.resolve(dirName, '../extras/basic_example/samplertcservice.js');
+var sampleServiceFile = path.resolve(dirName, '../apps/current_app/samplertcservice.js');
```

Comment:
Is it OK to use "main.js" instead of "samplertcservice.js" here?

Reply:
Yes, this should be changed thanks.